data/json/starting_missions.json

```diff
@@ -627,6 +627,82 @@
     },
     "origins": [ "ORIGIN_GAME_START" ]
   },
+  {
+    "id": "MISSION_HELICOPTER_REFUEL",
```

Comment:
This new mission doesn't seem to be set neither for profession nor for scenario?

Reply:
This mission got mistakenly copy pasted from just above in the doc when I passed the changes to my pc, I need to delete it.